mora/service/org.py

```diff
@@ -142,7 +147,16 @@ def get_organisation(orgid):
     # 0.8s -> 12.3s for 28k users and 33k functions
     # https://redmine.magenta-aps.dk/issues/21273
     users = c.bruger(tilhoerer=orgid)
-    functions = c.organisationfunktion(tilknyttedeorganisationer=orgid)
+    engagements = c.organisationfunktion(tilknyttedeorganisationer=orgid,
+                                         funktionsnavn=keys.ENGAGEMENT_KEY)
+    associations = c.organisationfunktion(tilknyttedeorganisationer=orgid,
+                                          funktionsnavn=keys.ASSOCIATION_KEY)
+    leaves = c.organisationfunktion(tilknyttedeorganisationer=orgid,
+                                    funktionsnavn=keys.LEAVE_KEY)
+    roles = c.organisationfunktion(tilknyttedeorganisationer=orgid,
+                                   funktionsnavn=keys.ROLE_KEY)
+    managers = c.organisationfunktion(tilknyttedeorganisationer=orgid,
+                                      funktionsnavn=keys.MANAGER_KEY)
```

Comment:
We'll have to keep our eye out for how this affects performance. In particular, do the SQL stored procedures perform the same array-compare when filtering for `funktionsnavn` as they do when filtering for `gyldighed`? If so, we can't afford this granularity, as the cost is prohibitive when counting the 36k engagements in the Aarhus spreadsheet.

Reply:
That's a good point. I'll let it stay for now, but keep in mind for next time we run it with a large dataset.
But let's also discuss later whether or not this level of granularity is actually useful information at all

Reply:
Yeah, the idea is just to present *something* — and let someone else decide whether it's useful at all…